x/tss/types/params.go

```diff
@@ -84,15 +84,15 @@ func DefaultParams() Params {
 				MinTotalShareCount:         4,
 				KeygenVotingThreshold:      utils.Threshold{Numerator: 15, Denominator: 20},
 				SignVotingThreshold:        utils.Threshold{Numerator: 11, Denominator: 20},
-				KeygenTimeout:              50,
-				SignTimeout:                50,
+				KeygenTimeout:              20,
+				SignTimeout:                20,
 			},
 		},
-		SuspendDurationInBlocks:          8646,
+		SuspendDurationInBlocks:          8500,
 		HeartbeatPeriodInBlocks:          50,
 		MaxMissedBlocksPerWindow:         utils.Threshold{Numerator: 5, Denominator: 100},
-		UnbondingLockingKeyRotationCount: 8,
-		ExternalMultisigThreshold:        utils.Threshold{Numerator: 3, Denominator: 6},
+		UnbondingLockingKeyRotationCount: 4,
+		ExternalMultisigThreshold:        utils.Threshold{Numerator: 4, Denominator: 8},
 		MaxSignQueueSize:                 50,
 		MaxSimultaneousSignShares:        26,
```

Comment:
we could increase `MaxSimultaneousSignShares` to ~100 or 150 to support multiple signing sessions